backend/filemanagement-service/src/main/java/hu/me/iit/malus/thesis/filemanagement/model/FileDescriptor.java

```diff
@@ -20,27 +16,33 @@
 @Getter
 @Setter
 @NoArgsConstructor
+@AllArgsConstructor
+@Builder()
 @ToString
 public class FileDescriptor {
```

Comment:
member-ek private-ok

Reply:
fel sem tűnt hogy ezek nem privatek :D lehet én rontottam el valamit a formázással vagy így volt régen is